releaseNotes/JavaScriptFeatureManagement.md

```diff
@@ -2,6 +2,28 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 2.0.0 - Jan 14, 2025
+
+### Enhancements
+
+This is the first stable release of the following features.
+
+* __Variant Feature Flags__
+
+  A variant feature flag is an enhanced feature flag that supports multiple states or variations. While it can still be toggled on or off, it also allows for different configurations, ranging from simple primitives to complex JSON objects. Variant feature flags are particularly useful for feature rollouts, configuration rollouts, and feature experimentation (also known as A/B testing).
+
+  The new `getVariant` method of `FeatureManager` has been introduced to evaluate the assigned variant based on the variant feature flag configuration and targeting context. [#13](https://github.com/microsoft/FeatureManagement-JavaScript/pull/13)
+
+* __Telemetry__
```

Comment:
```suggestion
* __Feature Evaluation Telemetry__
```

I think it might be better to be specific about the fact it's telemetry on feature evaluations.